examples/motomed_stim_example.py

```diff
@@ -1,7 +1,7 @@
 import time
 
-from pyScienceMode2.rehastim_interface import Stimulator as St
-from pyScienceMode2 import Channel as Ch
+from pyScienceMode2.rehastim_interface import Stimulator2 as St
+from pyScienceMode2 import channel as Ch
```

Comment:
The file channel must be in lower case and the class must be writing : Channel

Reply:
Done.